### PR TITLE
chore: remove old policy addon images

### PR DESCRIPTION
**Pull request by anlandu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Flagged for CVEs in 20221215 and 20220512 which are not used in >=1.24. Removing as 1.23 is EOL
**Which issue(s) this PR fixes**:

**Requirements**:
- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Review by ganeshkumar5699 — APPROVED** [review]

**Review by alexeldeib — APPROVED** [review]

**Comment by anlandu** [comment]
@alexeldeib Just to confirm, we've got about 15k clusters still on 1.23, for those that don't autoupgrade to 1.24 will they still have access to the old image? Thank you!

**Comment by alexeldeib** [comment]
yes, this is purely a caching mechanism, any cluster can use any images

**Comment by anlandu** [comment]
Holding off for one more person's sign off, my teammate will merge after that

**Review by robbiezhang — APPROVED** [review]

**Pull request merged**
